### PR TITLE
First playground

### PR DESCRIPTION
**Pull request by MiguelAskar**
First swift exercises ready (I hope) to be pulled.

### CODE REVIEW

File: BasicCoding/BasicCoding.xcodeproj/xcuserdata/miguel.askar.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cmorales-globant) [inline BasicCoding/BasicCoding.xcodeproj/xcuserdata/miguel.askar.xcuserdatad/xcschemes/xcschememanagement.plist]: We'll need to remove all this xcuserdata changes, they can be added to the .gitignore

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+//Exercise 2
+
+func comparator(_ array: [Int], _ number: Int)
+{
+    var greaterCounter = 0
+    var lowerCounter = 0
+    for item in array
+    {
```
Comment (cmorales-globant) [inline BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: We want to keep opening braces in line with the declaration that required it. For example

struct Foo {
}

func foo {
}

if (foo == 0) {
}

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        if item > number
+        {
+            greaterCounter += 1
+        }else if item < number
```
Comment (pablocalcagnino) [inline BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: also check spacing between the bracket and de "else"

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+let number2 = 101
+
+comparator(array, number2)
+
+
+//Exercise 3
+
+func printIfPositiveInteger(number: String) {
+  guard let value = Int(number), value >= 0  else
+  {
```
Comment (pablocalcagnino) [inline BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: brackets in same line as Camilo mentioned

**Review by pablocalcagnino — CHANGES_REQUESTED** [review]

File: BasicCoding/BasicCoding.xcodeproj/xcuserdata/miguel.askar.xcuserdatad/xcschemes/xcschememanagement.plist (hunk 1 above)
Comment (MiguelAskar) [reply BasicCoding/BasicCoding.xcodeproj/xcuserdata/miguel.askar.xcuserdatad/xcschemes/xcschememanagement.plist]: Fixed, gitignore fixed

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift (hunk 2 above)
Comment (MiguelAskar) [reply BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: Fixed, brackets corrected

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift (hunk 3 above)
Comment (MiguelAskar) [reply BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: Fixed. Space added after brackets

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift (hunk 4 above)
Comment (MiguelAskar) [reply BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift]: Brackets fixed.

File: BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+let distance: UInt = 41
+
+switch distance {
+    case 0:
+        print("Here")
+    case 0..<5:
+        print("Inmediate vicinity")
+    case 5...15:
+        print("Near")
+    case 15...40:
```
Comment (cmorales-globant) [inline BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift:14]: 15 is duplicated, already included in previous case.

**Pull request merged**